crates/ruff_python_formatter/src/context.rs

```diff
@@ -23,21 +23,21 @@ impl<'a> ASTFormatContext<'a> {
         }
     }
 
-    pub fn contents(&self) -> &'a str {
-        self.contents
-    }
-
-    pub fn locator(&self) -> Locator<'a> {
-        Locator::new(self.contents)
-    }
+    //pub(crate) fn contents(&self) -> &'a str {
+    //    self.contents
+    //}
+    //
+    //pub(crate) fn locator(&self) -> Locator<'a> {
+    //    Locator::new(self.contents)
+    //}
```

Comment:
Revert?

Reply:
the are unused so clippy complains. `#[allow(unused)]` would be an alternative

Reply:
You can make `format_range` `pub` :D (it should really be `pub`)

Reply:
i don't see any `format_range` function, i now switched to `#[allow(unused)]`